api/src/schema/directory-quick-facts.graphql

```diff
@@ -112,17 +95,19 @@ extend type Bacenta {
   avgWeekdayStats(days: Int!): Stats!
     @cypher(
       statement: """
-      MATCH (this)-[:HAS]->(fellowship) WHERE fellowship:Fellowship OR fellowship:ClosedFellowship
-      MATCH (fellowship)-[:HAS_HISTORY]-(:ServiceLog)-[:HAS_SERVICE]->(record:ServiceRecord)
-      MATCH (record) WHERE NOT (record:NoService)
-      MATCH (record)-[:SERVICE_HELD_ON]->(date:TimeGraph)
-      MATCH (date)
-      WHERE date.date > date() - duration({days: $days})
-      WITH sum(record.attendance) AS attendance, sum(record.income) as income, this, date(date.date).week as weeknumber
+      MATCH (this)
+      WITH date() as today, this
+      WITH date(today) - duration({days: 200}) AS startDate, this
```

Comment:
How come you took away the days argument?

Reply:
This is an over sight